packages/flutter_tools/test/application_package_test.dart

```diff
@@ -154,6 +186,117 @@ void main() {
   });
 }
 
+const String _aaptDataWithOnlyEnabledNotLauncherMainActivity =
```

Comment:
Can you replace this with actual XML?  I don't think this format is readable enough for general consumption.

Reply:
Also, please name all of these examples in the affirmative, for example:
_aaptDataWithEnabledActivityAndMainLauncherActivity

I'm not sure if the above name is appropriate for this example because I'm not entirely sure what you're showing here, but notice that the above name does not include any negatives.